trhift.go

```diff
@@ -0,0 +1,315 @@
+// Copyright (C) 2020 Storj Labs, Inc.
```

Comment:
the year here is wrong and the spelling of the filename is wrong

Reply:
IANAL, but the year seems to be good for me. The main motivation of using years (AFAIK) is defining the earliest date of the invention, to make it easier to compare it in case of prior-art research.

This code is from 2020, as I just coped it during refactor, I didn't really invented it.

But let me know if I am wrong. It costs 0 for me to update....

Reply:
If copied, then yes, 2020 is more appropriate.